src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -308,7 +308,7 @@ class CargoTestEventsConverter(
             if (parseResult == null) {
                 builder.addAttribute("message", "")
             } else {
-                val (message, diff) = parseResult
+                val (message, diff, _) = parseResult
```

Comment:
> the old line doesn't even compile

And it's the main difference (IMO). This feature was designed in this way on purpose. It's not a pattern matching (kotlin doesn't have this feature for now) only destructuring, so why I have to add useless `_` in all project when I just add a new field into my data class at the end that doesn't break API and even ABI (in case of destructuring, of course)?
I understand the initial purpose of this inspection - if I added new property in the middle of data class, it may break other code without compilation issues (when types match). But current warnings are more false-positive because there isn't any error in this code.

Note, in Rust you can use `..` to ignore the remaining part of struct, kotlin also doesn't have this feature.
Note (2), the corresponding inspection is disabled by default in IDE which is suggestive